src/components/DeleteHelicopter.jsx

```diff
@@ -0,0 +1,87 @@
+import { useEffect } from "react";
+import { useDispatch, useSelector } from "react-redux";
+import {
+  fetchHelicopters,
+  deleteHelicopter,
+  updateHelicopters,
+} from "../features/helicopters/helicopterSlice";
+import LayoutComponent from "../Layout";
+import { Button, Table, Popconfirm } from 'antd';
+import { QuestionCircleOutlined } from "@ant-design/icons";
+
+const DeleteHelicopterComponent = () => {
```

Comment:
- [ ] I still can't delete helicopters:

![image](https://github.com/emmanuelsteven/heli-hire-captsone-frontend/assets/57421823/7332fee3-a240-40c0-8f2d-50878515d0c5)

preview response:

![image](https://github.com/emmanuelsteven/heli-hire-captsone-frontend/assets/57421823/e0d8c141-0905-440b-9b1d-e8e47f3cd395)

payload:

![image](https://github.com/emmanuelsteven/heli-hire-captsone-frontend/assets/57421823/98fb5f46-3699-4f96-a0a0-3d9a63727bf8)

## Source of the issue and possible solution

The ":destroy" route for helicopters has been exempted:

![image](https://github.com/emmanuelsteven/heli-hire-captsone-frontend/assets/57421823/dfbf8775-611a-4c94-93b7-eb673554ecd0)

_This solution was tested._

Reply:
thanks for pointing it out!
fixed